mackerel-plugin-postgres/lib/postgres.go

```diff
@@ -365,6 +406,13 @@ func (p PostgresPlugin) GraphDefinition() map[string]mp.Graphs {
 				{Name: "temp_bytes", Label: "Temporary file size (byte)", Diff: true, Stacked: false},
 			},
 		},
+		"xlog_location": {
+			Label: (labelPrefix + " Amount of Transaction location change"),
+			Unit:  "integer",
```

Comment:
Prefer `Unit: "bytes"`

Reply:
Note that we don't mix metrics with different units, we should choose appropriate unit for each graph so that the values of the metrics are formatted properly.

Reply:
Ok, I understand that unit is set to bytes. But the units of other metrics are integers, not bytes. Should I adapt to your comment?

https://github.com/mackerelio/mackerel-agent-plugins/pull/476/files/06497c37ff94325d8b2703b72cb2a97d2b193ec5#diff-df6813af456003abfff9f4750fc44f6eR404

Reply:
Yes, if the value is collected in bytes.

Reply:
Fixed this by https://github.com/mackerelio/mackerel-agent-plugins/pull/476/commits/1109eeeeb5584d6d73deef85c96ac19083fe33d4.